app/src/main/kotlin/at/bitfire/davdroid/db/WebDavDocument.kt

```diff
@@ -27,7 +27,9 @@ import java.time.Instant
         ForeignKey(entity = WebDavDocument::class, parentColumns = ["id"], childColumns = ["parentId"], onDelete = ForeignKey.CASCADE)
     ],
     indices = [
-        Index("mountId", "parentId", "name", unique = true)
+        Index("mountId"),
+        Index("parentId"),
+        Index("name", unique = true),
```

Comment:
I think this would forbid the same file name ("README") to be present in different directories/mounts. We should keep the original index, if possible.

Above two foreign keys with `childColumn=mountId` and `childColumn=parentId` are defined. `ForeignKey` docs say:

> It is also recommended to create an index on the child table to avoid full table scans when the parent table is modified. If a suitable index on the child table is missing, Room will print RoomWarnings. MISSING_INDEX_ON_FOREIGN_KEY_CHILD warning.

So we need two indices:

- `mountId` (will be used to find the children when a `WebDavMount` is deleted) – this is already covered by the `Index("mountId", "parentId", "name", unique = true)` index because it's the leftmost column, and
- `parentId` (will be used to find the children when a `WebDavDocument` is deleted).

It should be enough to just add a normal index about `parentId`.